backend/pennclubs/settings/base.py

```diff
@@ -203,7 +203,8 @@
 APPLY_URL = "https://{domain}/club/{club}/apply"
 
 OSA_EMAILS = ["vpul-orgs@pobox.upenn.edu"]
-QUEUE_OPEN = False
+QUEUE_OPEN = True
+NEW_QUEUE_OPEN = False
```

Comment:
Nit: would be nicer to make these `APPROVAL_QUEUE_OPEN` and `REAPPROVAL_QUEUE_OPEN`.